docs/installation.md

```diff
@@ -28,75 +28,9 @@ Before getting started to develop a first adapter with SIP the following require
 
 ## Create the project
 
-### How to create a SIP Adapter from Maven Archetype
+### How to create a SIP Adapter using SIP Archetype
 
-In order to make setting up a new integration adapter project an easy task, we have created a dedicated SIP (Adapter) Maven archetype, that gives the ability to quickly bootstrap a new integration adapter, along with all needed Maven dependencies and goodies, as well as the recommended, preferred organization of the project.
+Guide for creating a new SIP Adapter from archetype can be found [here](https://ikor-gmbh.github.io/sip-framework/archetype/).
```

Comment:
This section now has strange structure. We have one title and subtitle and the content is only link to another page with description.
Perhaps we can make link from ### How to create a SIP Adapter using SIP Archetype, without any additional text. 
Maybe we change it a bit to: ### Creating SIP Adapter using SIP Archetype or something similar.

Reply:
I'd say using the title as a link wouldn't be very user friendly, due to titles already having color similar to anchors, while in regular paragraphs they stand out.

Reply:
I would prefer to keep it like this.